gst/nnstreamer/tensor_query/nnstreamer_edge_internal.c

```diff
@@ -86,31 +317,216 @@ nns_edge_get_handle (const char *id, const char *topic, nns_edge_h * edge_h)
    * 1. consider adding hash table or list to manage edge handles.
    * 2. compare topic and return error if existing topic in handle is different.
    */
-  eh = (nns_edge_handle_s *) malloc (sizeof (nns_edge_handle_s));
+  eh = g_try_new0 (nns_edge_handle_s, 1);
```

Comment:
line 89 and 95.
plz do not change this, we should remove glib dependency later.

Reply:
Changed. Thanks.